src/runtime_src/tools/xclbinutil/unittests/PSKernel/embedded_metadata_expected.xml

```diff
@@ -74,7 +74,7 @@
           <arg name="arg4" addressQualifier="0" id="4" size="0x8" offset="0x38" hostOffset="0x0" hostSize="0x8" type="int"/>
           <arg name="arg5" addressQualifier="0" id="5" size="0x4" offset="0x40" hostOffset="0x0" hostSize="0x4" type="float"/>
           <arg name="arg6" addressQualifier="1" id="6" size="0x10" offset="0x44" hostOffset="0x0" hostSize="0x10" type="float*"/>
-          <arg name="arg7" addressQualifier="1" id="7" size="0x10" offset="0x54" hostOffset="0x0" hostSize="0x10" type="xrtHandles*"/>
+          <arg name="arg7" addressQualifier="1" id="" size="0x10" offset="0x54" hostOffset="0x0" hostSize="0x10" type="xrtHandles*"/>
```

Comment:
I would recommend naming this argument for what it is, e.g. 'xrt_handles'.  This is but one reserved argument to the PS kernel, what if we come up with others, how are they distinguished?   Obviously 'type' can be used, but could there be two handles one day of same type?

For comparison, OpenCL kernels have a few rtinfo (i guess runtime info) arguments that are used by XRT implementation but clearly differentiated from index arguments.   I think we should model reserved arguments for PSKernel the same way.  Whether the reserved arguments are before or after numbered or even in the middle doesn't really matter.   The address qualifier doesn't matter either.
```
<arg name="group_id" addressQualifier="0" id="" port="S_AXI_CONTROL" size="0x4" offset="0x10" hostOffset="0x0" hostSize="0x4" type="uint" origUse="function"/>
<arg name="group_id" addressQualifier="0" id="" port="S_AXI_CONTROL" size="0x4" offset="0x18" hostOffset="0x4" hostSize="0x4" type="uint" origUse="function"/>
<arg name="group_id" addressQualifier="0" id="" port="S_AXI_CONTROL" size="0x4" offset="0x20" hostOffset="0x8" hostSize="0x4" type="uint" origUse="function"/>
<arg name="global_offset" addressQualifier="0" id="" port="S_AXI_CONTROL" size="0x4" offset="0x28" hostOffset="0x0" hostSize="0x4" type="uint" origUse="function"/>
<arg name="global_offset" addressQualifier="0" id="" port="S_AXI_CONTROL" size="0x4" offset="0x30" hostOffset="0x4" hostSize="0x4" type="uint" origUse="function"/>
<arg name="global_offset" addressQualifier="0" id="" port="S_AXI_CONTROL" size="0x4" offset="0x38" hostOffset="0x8" hostSize="0x4" type="uint" origUse="function"/>
<arg name="access1" addressQualifier="1" id="0" port="M_AXI_GMEM" size="0x8" offset="0x40" hostOffset="0x0" hostSize="0x8" type="char*"/>
```

Reply:
Chatted with Soren.  The current code as implemented will be used.